pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -1528,7 +1532,7 @@ func (dr *DashboardServiceImpl) getDashboardThroughK8s(ctx context.Context, quer
 		query.UID = result.UID
 	}
 
-	out, err := dr.k8sclient.Get(ctx, query.UID, query.OrgID, v1.GetOptions{}, subresource)
+	out, err := dr.k8sclient.Get(ctx, query.UID, query.OrgID, v1.GetOptions{}, latestSub, dtoSub)
```

Comment:
why are both latest and dto in the path?  i would expect one or the other

Reply:
Good question. I don't know enough about the `latest` subresource. The main aspect to look into is whether there is large object support in place for restoring dashboards. In other words, if a large object is getting restored, ensuring the rehydration is taking place. This sounds like something I need to understand for large object history. I can get back to you on this later and make them mutually exclusive for now.